neo/io/nixio.py

```diff
@@ -291,6 +291,10 @@ def _nix_to_neo_block(self, nix_block):
             # parent reference
             newchx.block = neo_block
 
+        # create object links
```

Comment:
why not using `create_relationship` instead of calling these two functions explicitely?

Reply:
Wasn't aware of that function (I was going off [the IO dev docs](https://neo.readthedocs.io/en/stable/io_developers_guide.html)).  Will update now.